train.py

```diff
@@ -110,323 +137,163 @@
 initialize_logging(model_logging_dir)
 logger.info("Using output dir: %s" % model_logging_dir)
 
-num_epochs = args["n_epochs"]
-val_per_n_steps = args["val_per_n_steps"]
-max_learning_rate = args["max_learning_rate"]
-min_learning_rate = args["min_learning_rate"]
-warmup_epochs = args["warmup_epochs"]
-weight_decay = args["weight_decay"]
-batch_size = args["batch_size"]
+parquet_file: str = args["parquet_file"]
+train_only_samples_file: str = args["train_only_samples_file"]
 
-# Default mask strategies and mask_ratio
-mask_strategies: Tuple[str, ...] = tuple(args["mask_strategies"])
-if (len(mask_strategies) == 1) and (mask_strategies[0] == "all"):
-    mask_strategies = MASK_STRATEGIES
-mask_ratio: float = args["mask_ratio"]
+dekadal = False
+compositing_window = "30D"
+if "10d" in parquet_file:
+    dekadal = True
+    compositing_window = "10D"
 
-parquet_file: str = args["parquet_file"]
-val_samples_file: str = args["val_samples_file"]
+valid_month_as_token = False
+if time_token != "none":
```

Comment:
is `time_token` sufficiently transparent in its meaning? I wouldn't expect it to be linked to `valid_month`.